Sources/FoundationEssentials/URL/URL.swift

```diff
@@ -2019,16 +2019,16 @@ extension URL {
             let third = utf8[thirdIndex]
             isAbsolute = (
                 first.isAlpha
-                && (second == UInt8(ascii: ":") || second == UInt8(ascii: "|"))
-                && third == UInt8(ascii: "\\")
+                && (second == ._colon || second == ._pipe)
+                && third == ._backslash
             )
 
             if !isAbsolute {
                 // Strip the drive letter so it's not mistaken as a scheme
                 filePath = String(filePath[thirdIndex...])
             } else {
                 // Standardize to "\[drive-letter]:\..."
-                if second == UInt8(ascii: "|") {
+                if second == ._pipe {
                     var filePathArray = Array(utf8)
                     filePathArray[1] = UInt8(ascii: ":")
                     filePathArray.insert(UInt8(ascii: "\\"), at: 0)
```

Comment:
Should we also use `._colon` and `._backslash` here?

Reply:
Oh, I like this idea!